ginicapture/src/doc/source/features.rst

```diff
@@ -0,0 +1,476 @@
+                e.printStackTrace();
+                handleFileImportError(e); // same method as in the Screen API example
+            }
+        }
+    }
+
+
+Help Screen Customization
+-------------------------
+
+You can show your own help screens in the Gini Capture SDK. You can pass the title and activity for each screen to the
+``GiniCapture.Builder`` using a list of ``HelpItem.Custom`` objects:
+
+.. code-block:: java
+
+    List<HelpItem.Custom> customHelpItems = new ArrayList<>();
+
+    customHelpItems.add(new HelpItem.Custom(R.string.custom_help_screen_title,
+            new Intent((Context) this, CustomHelpActivity.class)));
+
+    GiniCapture.newInstance()
+            .setCustomHelpItems(customHelpItems)
+            .build();
+
+You can also disable the supported formats help screen by passing ``false`` to
+``GiniCapture.Builder.setSupportedFormatsHelpScreenEnabled()``.
+
+Event Tracking
+--------------
+
+You have the possibility to track various events which occur during the usage of the Gini Capture SDK.
+
+To subscribe to the events you need to implement the ``EventTracker`` interface and pass it to the builder when creating
+a new ``GiniCapture`` instance:
+
+.. code-block:: java
+
+    GiniCapture.newInstance()
+        .setEventTracker(new MyEventTracker());
+        .build();
+
+In ``MyEventTracker`` you can handle the events you are interested in.
+
+.. code-block:: java
+
+    class MyEventTracker implements EventTracker {
+
+        @Override
+        public void onCameraScreenEvent(final Event<CameraScreenEvent> event) {
+            switch (event.getType()) {
+                case TAKE_PICTURE:
+                    // handle the picture taken event
+                    break;
+                case HELP:
+                    // handle the show help event
+                    break;
+                case EXIT:
+                    // handle the exit event
+                    break;
+            }
+        }
+
+        @Override
+        public void onOnboardingScreenEvent(final Event<OnboardingScreenEvent> event) {
+            (...)
+        }
+
+        @Override
+        public void onAnalysisScreenEvent(final Event<AnalysisScreenEvent> event) {
+            (...)
+        }
+
+        @Override
+        public void onReviewScreenEvent(final Event<ReviewScreenEvent> event) {
+            (...)
+        }
+
+    }
+
+.. note::
+
+     If you use the Screen API all events will be triggered automatically.
+
+     If you use the Component API some events will not be triggered (for ex. events which rely on
+     ``Activity#onBackPressed()``). You can check in the table below whether all the events you are interested in are
+     triggered.
+
+     To manually trigger events just call the relevant method of your ``EventTracker`` implementation with the required
+     event.
+
+Events
+~~~~~~
+
+Event types are partitioned into different domains according to the screens that they appear at. Each domain has a
```

Comment:
Not sure `at` is the right preposition here but maybe it is domain specific 🤷. I would use `on` or `in`.